scripts/momentOfTruthPostProcessing.js

```diff
@@ -65,7 +66,10 @@ try {
 }
 
 function compareJsonRef(beforeJsonRef, afterJsonRef) {
-    return (beforeJsonRef.replace(/json:\d+:\d+/, '') == afterJsonRef.replace(/json:\d+:\d+/, ''));
```

Comment:
What's the intent behind this change?

Reply:
This is related to the issue specified by Joel.